docs/Researcher/workloads/inference-overview.md

```diff
@@ -50,34 +51,36 @@ This has the benefit of conserving resources at the risk of a delay from "cold s
+When deploying models and running inference workloads, you may need to update the workload configuration in a live manner, without impacting the important services that are provided by the workload.
 
-This means that an ML engineer can submit updates to an existing inference workload whether it is currently running, pending (or any other status).
+This means you can submit updates to an existing inference workload whether it is currently running, pending, or any other status.
 
-Following are a few examples of updates that can be implemented:
+The following are a few examples of updates that can be implemented:
 
 * Changing the container image to deploy a new version of the model  
-* Changing different parameters (such as env variables)  
-* Changing the compute resources to improve performance  
-* Change the number of replicas and scale plan to adapt to requirement changes and scales
+* Changing different parameters (such as environment variables)  
+* Changing compute resources to improve performance  
+* Changing the number of replicas and scale plan to adapt to requirement changes and scales
 
-As stated above, during the update and until its successful completion, the service that the workload provides is not jeopardized as these are production-grade workloads. Hence consumers can continue using the model (send prompts for example) during the updating process.
+During the update and until its successful completion, the service that the workload provides is not jeopardized as these are production-grade workloads. Hence, consumers can continue using the model (send prompts for example) during the update process.
 
-During the update process of an inference workload, a new revision of pod(s) is created. This revision is the new desired specification of the workload. Although several updates can be submitted consecutively (even if the process of the previous update is not complete), the target goal (the desired specification) is always according to the last submitted update (the previous updates are ignored).
+During the update process of an inference workload, a new revision of pod(s) is created. This revision is the new desired specification of the workload. Although several updates can be submitted consecutively even if the process of the previous update is not complete, the target goal is always according to the last submitted update. This means, the previous updates are ignored.
 
-Once the new revision is created completely (according to the desired spec) and up and running, the entire traffic of requests is navigated to the new revision, and the original workload is terminated. Then the update process is considered complete.
+Once the new revision is created completely and up and running, the entire traffic of requests is navigated to the new revision, and the original workload is terminated. Then the update process is considered complete.
```

Comment:
It is still the same workload. so:
and the original workload is terminated --> and the original revision is terminated and the resource are back to the shared pool